tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
+envlist = py{39,310,311}
+isolated_build = true
+
+[testenv]
+deps = -r requirements/base.txt
+setenv =
+  PYTHONPATH = {toxinidir}{/}src
```

Comment:
hmm we have this `PYTHONPATH` thing in Scipp but we don't have it in Plopp and Scippnexus. Is it really needed?

Reply:
It shouldn't be. This is only used in scipp because it doesn't install the package because that takes too long. But here, `testenv` does install it. So the env var can go.